frontend/src/views/toolbox/clean/index.vue

```diff
@@ -641,12 +639,20 @@ function load18n(label: string) {
             return i18n.global.t('clean.dockerLog');
         case 'task_log':
             return i18n.global.t('clean.taskLog');
-        case 'shell':
-            return i18n.global.t('clean.shell');
         case 'containerShell':
             return i18n.global.t('clean.containerShell');
         case 'curl':
-            return i18n.global.t('clean.curl');
+        case 'shell':
+        case 'app':
+        case 'website':
+        case 'database':
+        case 'directory':
+        case 'log':
+        case 'cutWebsiteLog':
+        case 'clean':
+        case 'snapshot':
+        case 'ntp':
+            return i18n.global.t('cronjob.' + label);
         case 'container_images':
             return i18n.global.t('clean.images');
         case 'container_containers':
```

Comment:
The code appears to have several logical inconsistencies:

1. **Duplicate Case Statements**: The `case 'snapshot'` is duplicated without a comment explaining the purpose. This likely indicates an error that should be corrected.

2. **Redundant Cases**: Some cases appear redundant, such as `"container_shell"` and `"curl"`, which are also present under different labels but do not match each other exactly.

3. **Inconsistent Pattern Matching**: There seems to be pattern matching based on substrings of the label rather than strict matches (`label.includes()`). This might lead to unexpected behavior if the order or structure of strings changes.

### Suggested Optimizations and Improvements

1. **Remove Duplicated Case Statement**:
   ```javascript
   case 'snapshoot': // Note: Typo fixed here
       return i18n.global.t('clean.snapshot');
   ```

2. **Clean Up Redundancies**:
   - Remove `"container_shell"` and `"curl"` since they already cover similar functionalities with different prefixs (e.g., `$SHELL`).

3. **Use Strict Matches for Consistency**:
   If you want to strictly match against specific patterns, consider using regular expressions or more explicit string comparisons instead of substring matching.

Here's what the optimized version could look like:

```javascript
function load18n(label: string) {
    switch (label.toLowerCase()) { // Convert label to lowercase for consistent comparison
        case "cache":
            return i18n.global.t("clean.cache");
        case "snapshot_tm":
            return i18n.global.t("clean.snapshotTmp");
        case "snapshot_local":
            return i18n.global.t("clean.snapshotLocal");
        
        case "docker_log":
        case "nginx_access": 
            return i18n.global.t("cleanup.log.");
        
        case "task_log":
           return i18n.global.t("clean.taskLog");

        case "container_image":
            return i18n.global.t("clean.images");
        case "container_container":
            return i18n.global.t("clean.containers");

        default:
            console.error(`Unknown clean action: ${label}`);
            return null;
    }
}
```

This approach ensures clearer logic and eliminates unnecessary duplication while maintaining consistency.